emojos.py

```diff
@@ -25,13 +25,26 @@
 
 @app.route('/<domain>')
 def emojo(domain):
+    if request.args.get('filter_all','') == 'on':
```

Comment:
Can you run a style linter? I believe it will require a space after the comma here.